src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -264,6 +264,8 @@ private void ImportCall(ILOpcode opcode, int token)
             var method = (MethodDesc)_canonMethodIL.GetObject(token);
 
             _compilation.TypeSystemContext.EnsureLoadableMethod(method);
+            if ((method.Signature.Flags & MethodSignatureFlags.UnmanagedCallingConventionMask) == MethodSignatureFlags.CallingConventionVarargs)
```

Comment:
We already do this when RyuJIT is importing a call and we see a varag method. Scanner needs to match behaviors that RyuJIT is doing. This wasn't hit by the test but I did it for parity.

Reply:
Can you please point me to where that happens? I searched for CallingConventionsVarargs but could not find it...

Reply:
We do this after we cast it to RyuJIT's enum:

https://github.com/dotnet/runtime/blob/3b2883b097a773715ca84056885e0ca1488da36e/src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs#L891-L895